### ISSUE TITLE
Update latest version of vuedraggable on npm to 4.x

### ISSUE THREAD

**Issue by realityfilter** [SortableJS/vue.draggable.next#130]
With vue 3 being the current default is it possible to update the latest version of vuedraggable on npm  from 2.24.3 to 4.1.0 ?

**Comment by jzs11** [comment]
Please...

**Comment by christhofer** [comment]
Please...

It is starting to irritate me seeing this plugin keeps appearing on `npm outdated` result just because the latest is 2.24.3 not 4.1.0

**Comment by DaveSanders** [comment]
Yeah this threw us for a loop today after running npm outdated.  I'd say that now that Vue3 is the default, the package should be bumped to the V3 version and maybe the old one renamed.

**Comment by AnthonyRuelle** [comment]
Please @David-Desmaisons 
Can you resolve this problem ? it's very annoying

**Comment by MetaCrawler** [comment]
> Yeah this threw us for a loop today after running npm outdated. I'd say that now that Vue3 is the default, the package should be bumped to the V3 version and maybe the old one renamed.

+1 same here.

In case the problem is unclear, steps to reproduce:
1) add to package.json
```
"dependencies": {
    "vuedraggable": "^4.1.0"
}
```
2) run "npm install
3) run "npm outdated"
```
Package       Current  Wanted  Latest  Location                   Depended by  
vuedraggable    4.1.0   4.1.0  2.24.3  node_modules/vuedraggable  vue-frontend
```

**Comment by kydner** [comment]
same issues, when im run npm install vuedraggable or yarn add vuedraggable, the version is 2.x.x, on my project before latest is version 4.x.x
